pom.xml

```diff
@@ -26,7 +26,8 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <records-editor.yaml.file>${project.basedir}/src/main/resources/swagger.api/records-editor.yaml</records-editor.yaml.file>
 
-    <folio-spring-base.version>1.0.2-SNAPSHOT</folio-spring-base.version>
+    <folio-spring-base.version>1.0.2</folio-spring-base.version>
+    <folio-service-tools.version>1.6.0</folio-service-tools.version>
```

Comment:
I think we should not add this dependency because it brings a lot of rmb and vert.x dependencies.

Reply:
And also copy code that works with token header

Reply:
removed folio-service-tools dependency